lib/widgets/fields/alignment_field.dart

```diff
@@ -0,0 +1,155 @@
+
+import 'package:flutter/material.dart';
+
+class AlignmentField extends StatefulWidget {
+  final void Function(Alignment) onChanged;
+
+  const AlignmentField({
+    Key? key,
+    required this.onChanged,
+  }) : super(key: key);
+
+  @override
+  _AlignmentFieldState createState() => _AlignmentFieldState();
+}
+
+class _AlignmentFieldState extends State<AlignmentField> {
+  int selectedOption = 5;
+  int onHoverOption = 5;
+  bool isHover = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Container(
+          // height: 71,
+          width: 318,
+          decoration: BoxDecoration(
+            color: const Color(0xFF36343B),
+            borderRadius: BorderRadius.circular(8),
+            border: Border.all(
+              color: Colors.white,
+              width: 1,
+            ),
+          ),
+          child: Column(
+            children: [
+              Row(
+                children: [
+                  customRadioButton(1),
+                  customRadioButton(2),
+                  customRadioButton(3),
+                ],
+              ),
+              Row(
+                children: [
+                  customRadioButton(4),
+                  customRadioButton(5),
+                  customRadioButton(6),
+                ],
+              ),
+              Row(
+                children: [
+                  customRadioButton(7),
+                  customRadioButton(8),
+                  customRadioButton(9),
+                ],
+              ),
+            ],
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget customRadioButton(int index) {
```

Comment:
Convert  widget Function into Class Widget , also Change the name CustomRadioButton to AlignmentFieldItem